src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -12542,6 +12542,174 @@ def test_aks_azure_service_mesh_enable_disable(
             ],
         )
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=116847878&view=results) to validate the change. Please commit the corresponding recording file (find from pipeline artifact) to `src/aks-preview/azext_aks_preview/tests/latest/recordings` if the test passed.